lib/Models/Cesium.js

```diff
@@ -639,6 +644,7 @@ function pickObject(cesium, e) {
         result.error = 'An unknown error occurred while picking features.';
     });
 
+    result.features = result.features.filter(uniqueElements);
```

Comment:
Good point, thanks. Yes, I found that the AREMI generation/available capacity data was giving duplicate features.